Insert.sql

```diff
@@ -0,0 +1,114 @@
+INSERT INTO `Capability` VALUES (1, 'Artificial Intelligence');
+INSERT INTO `Capability` VALUES (2, 'Business Development and Marketing');
+INSERT INTO `Capability` VALUES (3, 'Operations');
+INSERT INTO `Capability` VALUES (4, 'Data');
+INSERT INTO `Capability` VALUES (5, 'Cyber Security');
+INSERT INTO `Capability` VALUES (6, 'Engineering');
+INSERT INTO `Capability` VALUES (7, 'Delivery');
+INSERT INTO `Capability` VALUES (8, 'People Team');
+INSERT INTO `Capability` VALUES (9, 'Product');
+INSERT INTO `Capability` VALUES (10, 'Testing');
+INSERT INTO `Capability` VALUES (11, 'Workday');
+INSERT INTO `Capability` VALUES (12, 'Commercial and Financial Management');
+INSERT INTO `Capability` VALUES (13, 'Experience Design');
+INSERT INTO `Capability` VALUES (14, 'Organisational Strategy and Planning');
+INSERT INTO `Capability` VALUES (15, 'Platforms');
+
+INSERT INTO JobFamily VALUES (1, 'AI Engineering', 1);
```

Comment:
Is there a reason some tables are surrounded with ticks (`) and others are not? It's not a deal-breaker - just might look cleaner if all consistent.